interface.css

```diff
@@ -483,3 +483,38 @@ textarea.form-control {
         transform: rotate(360deg);
     }
 }
+
+.colorpicker-2x .colorpicker-saturation {
+  width: 235px;
```

Comment:
@hcarneiro The sidebar changes size according to the window width, does it not?

Reply:
@tonytlwu Not anymore!